tutorials/e-commerce-store-codelab/step-01-setting-up-the-project.md

```diff
@@ -0,0 +1,293 @@
+---
+description: Getting started with a Remix app when building an e-commerce store.
+keywords:
+  - turso
+  - remix
+  - e-commerce
+---
+
+# Step 1: Setting up the project
+
+Before getting started, make sure that you satify the following pre-requisites:
+
+- You need the latest [LTS version of Node.js] to work with node packages.
```

Comment:
Is there a known minimum version required? It might be much to ask someone to always have the latest, since that's a moving target over time. Best if they can decide quickly if what they have now is sufficient.

Reply:
Used this as it always tends to be the best bet, and generally frameworks tend to keep up with it. It also future-proofs the tutorial.